reconcile/saas_auto_promotions_manager/merge_request_manager/metrics.py

```diff
@@ -16,6 +16,8 @@ class SAPMOpenedMRsCounter(SAPMBaseMetric, CounterMetric):
     "Counter for the number of opened auto-promotion MRs"
 
     is_batchable: bool
+    # We do not expect batches >10, i.e., cardinality will stay in check here.
+    batch_size: int
```

Comment:
do you still need the `is_batchable` label after introducing the `batch_size` one?

Reply:
Very good question. I do need it though, because a MR with batch size 1 can be both, batchable and non-batchable. 

The batchable attribute describes whether a specific hash has failed MR checks previously (i.e., was previously removed from a batched MR).

Reply:
ah i see. thanks for the insights